### ISSUE TITLE
Upgrade dependencies

### ISSUE THREAD

**Issue by ckenst** [TestingConferences/testingconferences.github.io#621]
Need to upgrade:

1. Ruby to 2.7.4
2. github-pages to latest (228)
3. nokogiri to latest
4. bundler version to latest
5. Jekyll to latest
6. Jekyll container image to match

**Comment by ckenst** [comment]
Need to split these up and only install dependencies as they make sense. Then match them to the docker versions.

**Comment by ckenst** [comment]
Blocked by #631
